docs-src/content/sessions/update.md

```diff
@@ -85,7 +85,17 @@ client.processText(token: token, text: text) { (sessionOutput) in
 {{< /tab >}}
 
 {{< tab "Java/Android" "java" >}}
-// Example coming soon!
+String text = "what's the weather forcast for today";
+TextInput txtMsg = TextInput.newBuilder()
+		.setText(text)
+		.build();
+SessionInput input = SessionInput.newBuilder()
+		.setToken(mToken)
+		.setText(txtMsg)
+		.build();
+SessionOutput resp = mDiathekeBlockingService.updateSession(input);
+mToken = resp.getToken();
+return resp;
 {{< /tab >}}
```

Comment:
This page is missing examples for [Process ASR Result](https://sdk-diatheke-pr-48.onrender.com/sessions/update/#process-asr-result), [Process Command Result](https://sdk-diatheke-pr-48.onrender.com/sessions/update/#process-command-result), and [Set Story](https://sdk-diatheke-pr-48.onrender.com/sessions/update/#process-command-result). I imagine they will all look very similar to the text example you have here, but we should show examples using the different message types.